templates/base/head_script.tmpl

```diff
@@ -41,6 +41,12 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 			copy_error: '{{.locale.Tr "copy_error"}}',
 			error_occurred: '{{.locale.Tr "error.occurred"}}',
 			network_error: '{{.locale.Tr "error.network_error"}}',
+			contributions: '{{.locale.Tr "contributions"}}',
+			no_contributions: '{{.locale.Tr "no_contributions"}}',
+			contributions_in_the_last_12_months: '{{.locale.Tr "contributions_in_the_last_12_months"}}',
+			on: '{{.locale.Tr "on"}}',
+			less: '{{.locale.Tr "less"}}',
+			more: '{{.locale.Tr "more"}}',
```

Comment:
I do not think it's good to fill these texts here. Only a few pages need them, and it's not clear what's they are for.

See the comment:

```
		{{/* this global i18n object should only contain general texts. for specialized texts,
 it should be provided inside the related modules by:
 (1) API response (2) HTML data-attribute (3) PageData */}}
```

Otherwise there will be a lot of texts in the future.

Reply:
Fixed